packages/supabase_flutter/test/supabase_flutter_test.dart

```diff
@@ -93,11 +94,32 @@ void main() {
 
   group('Deep Link with PKCE code', () {
     late final PkceHttpClient pkceHttpClient;
+    late final bool mockEventChannel;
+
+    /// Check if the current version of AppLinks uses an explicit call to get
+    /// the initial link. This is only the case before version 6.0.0, where we
+    /// can find the getInitialAppLink function.
+    bool appLinksExposesInitialLinkInStream() {
```

Comment:
Something like this?
```suggestion
    /// can find the getInitialAppLink function.
    /// 
    /// CI pipeline is set so that it tests both app_links newer and older than v6.0.0
    bool appLinksExposesInitialLinkInStream() {
```